### ISSUE TITLE
update getblobbers endpoint 

### ISSUE THREAD

**Issue by cnlangzi** [0chain/0chain#1032]
from https://github.com/0chain/blobber/issues/127

[/v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7/getblobbers](https://dev.0chain.net/miner01/v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7/getblobbers) should be updated  in the storagesc code

- change `used` name to `allocated`
- add `used` for storage that is actually used by the client

**Comment by peterlimg** [comment]
@cnlangzi Is there anyone working on this, API changes must be done before Mainnet, otherwise it would be painful to do so.

**Comment by ma2b0043** [comment]
@cnlangzi when can we expect to close this issue?

**Comment by guruhubb** [comment]
Check https://github.com/0chain/0chain/pull/1191#issuecomment-1106784333

But we need an automated way to report capacity on the chain when a disk is added to the server.
@cnlangzi @realrisman and @lpoli

**Comment by cnlangzi** [comment]
@realrisman  I will take over this issue, please asks new issue from @dabasov , thanks.

**Comment by cnlangzi** [comment]
blocked by https://github.com/0chain/0chain/issues/1115

**Comment by peterlimg** [comment]
@cnlangzi please help and fix this.
